tortoise/backends/asyncpg/client.py

```diff
@@ -172,7 +173,7 @@ def acquire_connection(self):
     async def commit(self):
         try:
             await self.transaction.commit()
-        except asyncpg.exceptions._base.InterfaceError as exc:
+        except (AttributeError, asyncpg.exceptions._base.InterfaceError) as exc:
```

Comment:
That was because the linter (correctly) complained that it was possible for `self.transaction` to not exist. Which is now forcefully set to None instead. AttributeError is if it is a None value.

I should probably make it clearer? Or is it perferred for it to raise a NPE?

Now that I think of it, a NPE would point out that something happened that should never be able to happen...